projects/observability/src/shared/dashboard/widgets/charts/cartesian-widget/cartesian-widget-renderer.component.ts

```diff
@@ -58,13 +58,17 @@ export class CartesianWidgetRendererComponent<TSeriesInterval> extends Interacti
     return this.model.getDataFetcher().pipe(
       tap(fetcher => {
         this.fetcher = fetcher;
+        const defaultInterval = this.model.defaultInterval?.getDuration();
 
         if (this.intervalSupported()) {
           this.intervalOptions = this.buildIntervalOptions();
-          this.selectedInterval = this.getBestIntervalMatch(this.intervalOptions, this.selectedInterval);
+          this.selectedInterval = this.getBestIntervalMatch(
```

Comment:
With this, if `defaultInterval` is not part of one of the supported options then it would choose AUTO. I think this is acceptable for now.